client/sites-dashboard/components/sites-dashboard-opt-in-banner.tsx

```diff
@@ -71,12 +72,18 @@ export const SitesDashboardOptInBanner = ( { sites }: SitesDashboardOptInBannerP
 		return null;
 	}
 
-	const handleAccept = () => {
-		setLandingPagePreference( {
+	const handleAccept = async () => {
+		await setLandingPagePreference( {
 			useSitesAsLandingPage: true,
 			updatedAt: new Date().valueOf(),
 		} );
 		dispatch( recordTracksEvent( 'calypso_sites_dashboard_landing_page_banner_accept_click' ) );
+		dispatch(
+			successNotice( __( 'Default page updated' ), {
```

Comment:
I'd say 
```suggestion
			successNotice( __( 'Landing page successfully updated' ), {
```

is a better copy.